devops/scripts/benchmarks/benches/umf.py

```diff
@@ -40,96 +41,44 @@ def benchmarks(self) -> list[Benchmark]:
             GBenchUmfProxy(self),
             GBenchJemalloc(self),
             GBenchTbbProxy(self),
+            GBenchMemoryOverhead(self),
```

Comment:
Correct me if I'm wrong, but memory overhead/fragmentation is simply an additional column in existing benchmark scenarios, right?
If that's the case, I don't think you want to add another benchmark since that will run the same scenario twice, and simply extract a different value.
Instead, you probably should return two results from a single benchmark. With different unit and result value.